test/loading.js

```diff
@@ -50,59 +51,64 @@ async function setTestCase() {
 
 describe('Running Pupeeteer', function() {
   describe('Loading Test Suite', function() {
-    it('Checking Scenes', async (done) => {
+    it('Checking Scenes', async done => {
       let error = false;
       const appTester = new LoadTester({
         slowMo: 0,
         host: 'http://localhost:3000',
-      })
+      });
 
-      require('child_process').exec('git rev-parse HEAD', function(err, stdout) {
+      require('child_process').exec('git rev-parse HEAD', function(_err, stdout) {
         console.log('Last commit hash on this branch is:', stdout);
         appTester.addStatErr('HASH', stdout);
       });
 
-      await setTestCase();
+      if (process.env.CACHEGENERATOR) {
```

Comment:
I don't think this is a good way to bootstrap vite cache. There must be a way to do it without booting a browser...

Reply:
Removed will be added in the next PR.